.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+    mocha: true
+  },
+  extends: ['standard'],
```

Comment:
is prettier configuration inside standard?

Reply:
I think yes, because we don't use `prettier` here but `prettierx` which is configured to perform like `standard`

I already ran `npm run format` on this PR, so you can see the diff what changes when performing npm run format + npm run lint